run-webots.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+webots --mode=fast --batch /app/worlds/soccer.wbt
```

Comment:
Just out of curiosity, what does `--batch` do?

Reply:
Not ask any blocking questions, if it were to ask you someting it will just always go with the default. I don't think it does anything here, but better safe than sorry if we change something and it starts to.

Reply:
Makes sense -- thanks @dbscoach !